packages/x-components/CHANGELOG.md

```diff
@@ -1,25 +1,22 @@
 # Change Log
 
-All notable changes to this project will be documented in this file.
-See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+All notable changes to this project will be documented in this file. See
```

Comment:
question: did prettier break?

Reply:
I'm not sure what happened here, seems the formatting was wrong somehow